foundationsync/indico_foundationsync.py

```diff
@@ -231,8 +239,19 @@ def fetch_rooms(self, connection, room_name=None):
 
             # Update room data
             room = self._update_room(room, room_data, room_attrs)
+            # Remove all existing full_access acl entries before adding the new ones
+            # otherwise changing the room owner/manager-group wont remove the old one
+            for manager in room.get_manager_list():
+                room.update_principal(manager, full_access=False)
+            if room_data.get('owner') is not None:
+                room.update_principal(room_data['owner'], full_access=True)
+            manager_group = room_attrs.get('manager-group')
+            if manager_group is not None:
+                group = GroupProxy(manager_group, provider='cern-ldap')
```

Comment:
let's log a warning if `group.group is None` (that means the group does not exist in ldap)